### ISSUE TITLE
Highcharts accessibility for event click() is missing

### ISSUE THREAD

**Issue by alu-ms** [highcharts/highcharts#20729]
#### Expected behaviour
Click event should have the same `event` context.

#### Actual behaviour
If you click on the point with a mouse you get a different result than with pressing enter on the keyboard.

#### Live demo with steps to reproduce
[demo](https://jsfiddle.net/BlackLabel/x091h5pb/show)
[SO thread](https://stackoverflow.com/questions/78071398/highcharts-accessibility-enter-and-click-events-function-mouseover-is-not-enoug)

#### Product version
Highcharts JS v11.3.0 (2024-01-10)

**Comment by karolkolodziej** [comment]
Hi @alu-ms! 

You are right the context of the `event` is different if you use the mouse and keyboard click- [demo](https://jsfiddle.net/BlackLabel/x091h5pb/show).
